department.rb

```diff
@@ -0,0 +1,37 @@
+# require './employee'
+
+class Department
+attr_reader :name, :roster
+  def initialize(name)
+    @name = name
+    @roster = []
+    @employees = []
+  end
+
+  def add_employee(employee)
+    @roster << employee
+  end
+
+  def total_salary
+    total = 0
+    @roster.each do |employee|
```

Comment:
Good logic.  As an alternative, you could do all of this with one `reduce` line.